nix/proto

```diff
@@ -29,5 +29,5 @@ case $command in
   allow) removeAll $1; add allowed $1;;
   ignore) removeAll $1; add ignored $1;;
   forget) removeAll $1;;
-  *) jq . protocols.json
+  *) jq . ../protocols.json
 esac
```

Comment:
Now that `protocols.json` is used not only in nix-way, I think this file should be moved outside of `nix/` as well. AFAIU it's an essential part of the main workflow and we want the main workflow to be possible w/o `nix`.

AFAIU this script doesn't really need `nix`, but it has some traces of `nix`. It essentially declares its dependencies (`jq` and `moreutils`) using `nix`. AFAIU in nix-less world it should have `#!/usr/bin/env bash` shebang and implicitly assume that `jq` and `moreutils` (specifically `sponge`) are installed. Maybe if we remove that `nix-shell` thing, the nix-way will break, but I hope it can be fixed so that both ways are supported.

Btw, I have found this issue https://github.com/stedolan/jq/issues/105 which is almost 7 years old, it's kinda funny that it has some many reactions and is still open and debatable :)

I would put this `proto` script to `scripts/proto` (since we have the `scripts/` folder already).

P. S. You will need to update `CONTRIBUTING.md`.